packages/ckeditor5-engine/docs/framework/deep-dive/schema.md

```diff
@@ -320,16 +329,25 @@ Here is a table listing various model elements and their properties registered i
 			<td class="value_positive_inherited"><code>true</code><a href="#inherited3"><sup>[3]</sup></a></td>
 		</tr>
 		<tr>
-			<td><code>listItem</code></td>
```

Comment:
This is not legacy, this is still used in lists when the `list.multiBlock` config option is set to `false`.

Reply:
https://github.com/ckeditor/ckeditor5/blob/39c7c09cc0adf20110cce5da79ed62aa0a4e2d7c/packages/ckeditor5-list/src/list/listediting.ts#L156-L165